### ISSUE TITLE
[feature request] Starting day of week

### ISSUE THREAD

**Issue by roidudev** [chronotruck/vue-ctk-date-time-picker#78]
I would like to set the weeks to start from Monday instead of Sunday in the calendar. Couldn't find a way to do it.

**Comment by LouisMazel** [comment]
But I added this  in 'upcomming features' in the README

**Comment by roidudev** [comment]
Thanks for the response.

We have moment's locale set to 'fi', so Monday should be the first day of the week. And in the component where we're using the date time picker, if we log the locale and firstDayOfWeek to console, it seems to be fine:

```
console.log(moment.locale()
// 'fi'
console.log(moment.localeData(moment.locale()).firstDayOfWeek())
// 1
```

But still the calendar shows weeks starting from Sunday. All the texts in the calendar are translated to Finnish though, so I'm guessing it's getting the correct locale.

**Comment by LouisMazel** [comment]
@roidudev 
Hi,


I released a beta version with this feature (2.0.0-rc4)
You can test it :
`npm install vue-ctk-date-time-picker@beta`
DEMO : https://htmlpreview.github.io/?https://github.com/chronotruck/vue-ctk-date-time-picker/blob/feature/refont/demo/index.html
DOC : https://github.com/chronotruck/vue-ctk-date-time-picker/blob/feature/refont/README.md

Le me know if you have others feedbacks (events, bugs, etc)

Thanks

**Comment by roidudev** [comment]
@LouisMazel 
Seems to work fine with all other first-day-of-week values but not with value 1 (Monday). When given value of 1, the date numbers in the calendar are positioned correctly, but the weekday header starts from Tuesday, see screenshot below:

![image](https://user-images.githubusercontent.com/45820301/50767238-be590980-1284-11e9-9d44-de053f10fd28.png)

**Comment by LouisMazel** [comment]
#105

I will released this soon in `2.0.0-rc7`

**Comment by LouisMazel** [comment]
I released : https://github.com/chronotruck/vue-ctk-date-time-picker/releases/tag/2.0.0-rc7
